packages/medusa/src/types/modules.ts

```diff
@@ -0,0 +1,4 @@
+export type ModulesResponse = {
```

Comment:
self-review: I couldn't agree with myself around what would be reasonable to respond with in the store. Decided to go with this; name of module + resolution path. Please feel free to pitch in @srindom

Reply:
Maybe that it should just be the `ModuleDefinition`?

Reply:
This is the response from `GET /admin/store` so I followed around general convention around suffixing these types with `Response`